pkg/apiserver/user/sso/service.go

```diff
@@ -254,16 +264,21 @@ func (s *Service) revokeAllImpersonations() error {
 }
 
 type oidcWellKnownConfig struct {
-	Issuer        string `json:"issuer"`
-	AuthURL       string `json:"authorization_endpoint"`
-	TokenURL      string `json:"token_endpoint"`
-	UserInfoURL   string `json:"userinfo_endpoint"`
-	EndSessionURL string `json:"end_session_endpoint"`
+	Issuer                           string   `json:"issuer"`
+	AuthURL                          string   `json:"authorization_endpoint"`
+	TokenURL                         string   `json:"token_endpoint"`
+	UserInfoURL                      string   `json:"userinfo_endpoint"`
+	EndSessionURL                    string   `json:"end_session_endpoint"`
+	JWKSURI                          string   `json:"jwks_uri"`
+	ResponseTypesSupported           []string `json:"response_types_supported"`
+	SubjectTypesSupported            []string `json:"subject_types_supported"`
+	IDTokenSigningAlgValuesSupported []string `json:"id_token_signing_alg_values_supported"`
```

Comment:
These fields are required according to the OIDC specification. https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata

Reply:
Could you help verify that newly added checks works for Okta? We have production users using Okta and when we adding more checks we need to make sure not breaking them.

Reply:
Yep, both checked.